scripts/install-worker.sh

```diff
@@ -203,7 +203,11 @@ EOF
 
 sudo yum install -y device-mapper-persistent-data lvm2
 
-INSTALL_DOCKER="${INSTALL_DOCKER:-true}"
+if [[ -z "$INSTALL_DOCKER" ]]; then
```

Comment:
Did you try building a 1.25 AMI?

Reply:
Yep, I built both 1.24 and 1.25, both succeeded. Launched instances with both and verified that 1.24 had `docker`, and 1.25 didn't.